src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball randomBall = new Ball();
+        String randomColor = ColorSupplier.getRandomColor();
+        Random random = new Random();
+
+        randomBall.setColor(randomColor.toString());
+
+        randomBall.setNumber(random.nextInt(100) + 1);
+
+        return randomBall;
```

Comment:
```suggestion
public class Lottery {
    private static final Random random = new Random();
        
    public Ball getRandomBall() {
        Ball randomBall = new Ball();
        String randomColor = ColorSupplier.getRandomColor();

        randomBall.setColor(randomColor);
        randomBall.setNumber(random.nextInt(100));

        return randomBall;
```
create global variable for Random
randomColor is already a String